requirements.txt

```diff
@@ -26,7 +26,6 @@ django-grappelli==2.8.1
 django-libsass==0.6
 django-redis==4.4.2
 -e git+https://github.com/kcsry/django-simple-email-confirmation.git@2df833cb3736264aac46f661c48bc2abdcbaa18f#egg=django-simple-email-confirmation
-django-zurb-foundation==5.5.0
```

Comment:
Because I saw bootstrap was used in the sass and it didn't make sense to
use both. Now that I know that we are not switching to bootstrap, I will
put it back in.